content/en/security/guide/security-findings-migration.md

```diff
@@ -0,0 +1,117 @@
+
+### Code Security Findings
+
+- Library Vulnerabilities (SCA)
+- Static Code Vulnerabilities (SAST)
+- Secrets
+- Runtime Code Vulnerabilities (IAST)
+- Infrastructure as Code
+
+### Cloud Security Findings
+
+- Misconfigurations (CSPM)
+- Host & Container Vulnerabilities
+- Identity Risks (CIEM)
+- Attack Paths
+
+API security findings are also produced by App & API Protection (in preview).
+
+## What is changing
+
+### New querying syntax
+
+Following this change, all Security Findings start being queried the same way, using shared syntax and attributes organized around namespaces. Here are some practical examples:
+
+| Before | After |
+|--------|-------|
+| **Misconfigurations:** `@workflow.triage.status:open status:critical`<br>**Library vulnerabilities:** `status:open severity:Critical` | **All findings:** `@status:open @severity:critical` |
+| **Misconfigurations:** `@dd_computed_attributes.is_publicly_accessible:true`<br>**Host Vulnerabilities:** `is_publicly_accessible:Accessible` | **All findings:** `@risk.is_publicly_accessible:true` |
+| **Library Vulnerabilities:** `library_name:org.apache.logging.log4j`<br>**Host Vulnerabilities:** `package:org.apache.logging.log4j` | **All findings:** `@package.name:org.apache.logging.log4j` |
+
+View the full specification here: {insert link}
+
+Every interface that allows querying security findings data gradually starts using the new data model:
+
+- Explorers, dashboards, notification rules and automation pipelines will use the new syntax
+- Workflow Automation
+- API & Terraform resources
+
+### New features
+
+Starting January 2026, the following features are rolled out using the new data model:
+
+- A new unified findings public API
+- Dashboard support for Code Security
+- Graphing Security Findings in Datadog Sheets
+- Datadog Workflow Automation support for all finding types
+- Using SQL to query Security Findings and join them with other Datadog Telemetry via DDSQL
+
+With more coming soon:
+
+- Data Access Controls
+- Datadog Security MCP
+- API & Terraform support for automation pipelines
+
+## When is it changing
+
+Products will gradually be onboarded to the new data model:
+
+{{< img src="security/schema-milestones.png" alt="Security Findings migration timeline showing Milestone 1 in January 2026 and Milestone 2 in April 2026" width="100%">}}
+
+### Milestone 1: January 2026
+
+**Migrated Findings** (use new schema):
+- Misconfigurations
+- Identity Risks
+- Attack Paths
+- IaC & API Security Findings
+
+**Platform Updates:**
+- Dashboards
+- DDSQL
+- Sheets
+- Findings Public API
+
+### Milestone 2: April 2026
+
+**Remaining Findings** (use new schema):
+- Cloud Security Vulnerabilities
+- Code Security Findings (SCA, SAST, IAST, Secrets)
+
+## How to prepare
+
+- Every in-app feature will be automatically migrated.
+- Legacy API endpoints and Terraform resources will eventually be deprecated.
```

Comment:
These aren't instructions on how to prepare. Can we rename the section or work the bullet points into the rest of the content on the page?